cmd/weaver/types.go

```diff
@@ -23,6 +25,27 @@ type argument struct {
 	ArrayLength    int // Set as 0 if not array
 }
 
+type procInfo struct {
+	Pid  uint32 `json:"pid,omitempty"`
+	Ppid uint32 `json:"ppid,omitempty"`
+	Comm string `json:"comm,omitempty"`
+}
+
+// UnmarshalBinary for procInfo
+func (i *procInfo) UnmarshalBinary(data []byte) error {
```

Comment:
Oh can you also make this `unmarshalBinary()` ?

Reply:
done